### PR TITLE
Fix world light config

### PR DESCRIPTION
**Pull request by ramon-bernardo**
### Pull Request Prelude
- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- Move world light config to game core

**Issues addressed:**
https://github.com/otland/forgottenserver/issues/4621

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by omarcopires** [comment]
we had everything centralized in config.lua, now we have several scripts with several different configurations, this is a great return.

**Comment by Sajgon** [comment]
Why is the config of defaultWorldLight even a thing? there's nothing to handle it being false

**Comment by Sajgon** [comment]
I think there's another issue with world light

Here it's returned as "worldLightLevel, worldLightColor"
```
	function Game.getWorldLight()
		return worldLightLevel, worldLightColor
	end
```

but on other places it's fetched as:
`local worldLightColor, worldLightLevel = Game.getWorldLight()`

File: data/lib/core/game.lua [hunk 1]
```diff
@@ -189,6 +189,12 @@ function Game.getUnpromotedVocations()
 end
 
 do
+	-- World Light
+	-- NOTE: if defaultWorldLight is set to true the world light algorithm will
+	-- be handled in the sources. set it to false to avoid conflicts if you wish
+	-- to make use of the function Game.setWorldLight(color, level)
```
Comment (amatria) [inline data/lib/core/game.lua:195]: Is this comment still true? Is it handled in the sources or is it handled in Lua? `data/scripts/globalevents/world_light.lua` seems to be doing just that.

**Review by amatria — APPROVED** [review]
LGTM

**Comment by amatria** [comment]
> we had everything centralized in config.lua, now we have several scripts with several different configurations, this is a great return.

You are absolutely right. It is a pity this parameter is now decentralized.

**Comment by MillhioreBT** [comment]
If it will no longer be a configuration then make the variable local, since it will only be used in that specific place, if someone changes it to false they know they have to create their own lighting system and they do not have to check if it is false or true elsewhere

Honestly, it doesn't bother me that this boolean is in the Lua configuration, it can be accessed from anywhere and it also doesn't litter the global environment with global variables.

**Comment by ramon-bernardo** [comment]
Similar https://github.com/otland/forgottenserver/pull/4790

**Pull request closed without merging**